packages/compass-connections-navigation/src/connections-navigation-tree.tsx

```diff
@@ -397,7 +397,7 @@ const ConnectionsNavigationTree: React.FunctionComponent<
         onDatabaseExpand(connection.connectionInfo.id, activeNamespace, true);
       }
     }
-  }, [activeNamespace, onDatabaseExpand, connections[0]?.connectionInfo.id]);
+  }, [activeNamespace, onDatabaseExpand, connections]);
```

Comment:
this bit is removed in https://github.com/mongodb-js/compass/pull/5734/files

Reply:
that should be fine though - right? can be resolved when merging?

Reply:
hopefully, just saying in case you see a conflict :)

Reply:
yea, I saw the same bit in your PR and had the same premonition 😄